author: Andres Salomon <dilinger@debian.org>
description: hardcode lld for whatever version of clang we're using

Upstream doesn't allow overridding the linker name (other than toggling
lld vs gold).

--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1048,7 +1048,7 @@ config("linker") {
       ldflags += [ "--ld-path=" +
                    rebase_path(experimental_linker_path, root_build_dir) ]
     } else if (use_lld) {
-      ldflags += [ "-fuse-ld=lld" ]
+      ldflags += [ "-fuse-ld=lld-19" ]
       if (lld_path != "") {
         ldflags += [ "-B" + rebase_path(lld_path, root_build_dir) ]
       }
